spynnaker/pyNN/models/neural_projections/delayed_machine_edge.py

```diff
@@ -39,6 +40,24 @@ def __init__(
             pre_vertex, post_vertex, label=label, traffic_weight=weight,
             app_edge=app_edge)
         self.__synapse_information = synapse_information
+        self.__undelayed_edge = None
+
+    @property
+    def undelayed_edge(self):
+        """ Get the edge used for Projections without extended delays
+
+        :rtype: ProjectionMachineEdge or None
+        """
+        return self.__undelayed_edge
+
+    @undelayed_edge.setter
+    def undelayed_edge(self, undelayed_edge):
```

Comment:
apart from the fact that you can change it at any point, and yet changing it means other things that have asked are wrong now. and given there shouldnt be any reason to cahnge it. id propose doing a null check to stop us doing stupid things

Reply:
The issue is then that if you reset and need to repartition, this will now fail...  It needs to be possible to set it more than once.  Given that the only code that sets it is self contained, I can't see how it would go wrong...